.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses. -->
+Addresses issue #
+
+## Checklist
+
+- [ ] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
+- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
+- [ ] My code follows the [coding standards](https://github.com/ampproject/amp-wp/contributing/engineering.md#coding-standards).
```

Comment:
This isn't really necessary because linting automatically checks it.

Reply:
Unless PR creator hasn't set this up locally, which this line reminds them of.

Reply:
We want to encourage folks reading the code standards. I am hope that we will extend it beyond just code formatting which is only a small piece of the puzzle. If you guys strongly feel it is unnecessary, happy to remove it.